try:
    print("x")
except:
    print("Coś poszło nie tak")
finally:
    print("Klauzula 'try except' jest zakończona")